setup.py

```diff
@@ -44,9 +44,8 @@
 
     install_requires=[
         "eth-hash>=0.1.0,<1.0.0",
-        "eth-utils>=1.0.1,<2.0.0",
+        "eth-utils>=1.3.0b0,<2.0.0a0",
```

Comment:
@Bhargavasomu Sorry for the delay but `eth_utils` ` 1.3.0` is now out. Can you change this to `1.3.0,<2.0.0`